test.js

```diff
@@ -1,12 +1,58 @@
+/* globals describe, it */
+var sleep = require('.');
+var assert = require('assert');
 
-var sleep = require('./');
+function assertApproxEqual(val1, val2) {
+  var epsilon = 1; // we require accuracy to the nearest millisecond
```

Comment:
You may need to increase this value. On my system, this value seems to work. If I recall, the `sleep()` system call doesn't always sleep for *exactly* the amount specified, so that's why this is here.